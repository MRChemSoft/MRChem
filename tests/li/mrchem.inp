# vim:syntax=sh:

abs_prec = 1.0e-3                 # Final precision in energy

MRA {
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
multiplicity = 2
$coords
Li   0.0     0.0      0.0
$end
}

WaveFunction {
  method = HF                     # Wave function method (HF or DFT)
  restricted = false
}

Properties {
  scf_energy = true               # Compute ground state energy
}

SCF {
  kain = 3                        # Length of KAIN iterative history
  rotation = 5                    # Number of iterations between diag/loc
  initial_guess = GTO             # Type of initial guess (none, gto, mw)
}
