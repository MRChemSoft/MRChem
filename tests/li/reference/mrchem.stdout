


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. ed8bfcd)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : 093397a

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 5
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 16.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -16.000000  -16.000000  -16.000000 ]
 upper bounds     = [   16.000000   16.000000   16.000000 ]
 total length     = [   32.000000   32.000000   32.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     Li           3.00000e+00   1.00000e-03   2.61594e-03
============================================================


============================================================
                    Initialize DFT grid
------------------------------------------------------------
 Nr  Element                        nNodes          nPoints
------------------------------------------------------------
  0     Li                             328           497664
------------------------------------------------------------
                 Wall time: 4.94499e-04 sec 
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    a    1                       9.999999988178128e-01
    1    a    1                       1.000000048121802e+00
------------------------------------------------------------
                 Wall time: 1.56579e-01 sec 
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    b    1                       9.999999988146178e-01
------------------------------------------------------------
                 Wall time: 8.72788e-02 sec 
============================================================


============================================================
 OrbitalVector:   3 orbitals     3 occupied     3 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       9.999999988178e-01    a   1    1.00000e+00
   1     0       1.000000048122e+00    a   1    1.00000e+00
   2     0       9.999999988146e-01    b   1    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     840      9.39575e-02
 Coulomb density                       328      8.60657e-02
 Coulomb potential                     200      9.70952e-01
 Coulomb grid size                     328           566784
 Coulomb grid change                    64           110592
 XC alpha density                      328      6.17052e-02
 XC beta density                       328      3.94738e-02
 XC evaluate xcfun                    2952      3.89474e+00
 XC energy                             328      3.23570e-05
 XC potential                          656      5.72360e-02
 XC grid size                          328           497664
 XC grid change                         64            96768
 Hartree-Fock exchange                1112      4.03675e+00
------------------------------------------------------------
                 Wall time: 9.28828e+00 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              2.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        1.30388e-03
 Diagonalizing matrix                           4.09600e-06
 Rotating orbitals                              1.25739e-02
------------------------------------------------------------
                 Wall time: 1.39291e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.413899258031377e+00
------------------------------------------------------------
 Total energy                        -7.413899258031377e+00
------------------------------------------------------------
                 Wall time: 4.88418e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -2.02696e+00
 Creating operator with lambda                 -1.39753e-01
 Creating operator with lambda                 -2.01683e+00
------------------------------------------------------------
                 Wall time: 1.62868e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 4.87733e-01
 Matrix part                                    6.91800e-06
------------------------------------------------------------
   0       1.49695e+00    584            0.00000e+00      0
   1       2.67528e-01    712            0.00000e+00      0
   2       1.49708e+00    392            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 5.00752e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00362e+00   328    0.00000e+00     0   1.84093e+00
  1    9.97808e-01   264    0.00000e+00     0   1.65654e+00
  2    1.00471e+00   328    0.00000e+00     0   1.64447e+00
------------------------------------------------------------
                 Wall time: 5.14204e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.25100e-06
 Copy orbitals                                  5.93052e-03
 Copy orbital updates                           7.41354e-03
------------------------------------------------------------
                 Wall time: 1.33991e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     840      8.97130e-02
 Coulomb density                       392      9.09247e-02
 Coulomb potential                     200      9.91273e-01
 Coulomb grid size                     392           677376
 Coulomb grid change                     0                0
 XC alpha density                      392      6.37610e-02
 XC beta density                       392      4.08231e-02
 XC evaluate xcfun                    3528      4.65611e+00
 XC energy                             392      2.50740e-05
 XC potential                          784      7.14661e-02
 XC grid size                          392           594432
 XC grid change                          0                0
 Hartree-Fock exchange                1112      3.91303e+00
------------------------------------------------------------
                 Wall time: 9.96778e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.06036e+00   3.76277e-02        328    a    1    0
  1  -1.41622e-01   4.44108e-02        264    a    1    0
  2  -2.05047e+00   3.85486e-02        328    b    1    0
------------------------------------------------------------
 Total error:                            6.98151e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -4.183542352258243e+00    -4.18354e+00    1
 Kinetic       7.370996344219272e+00     7.37100e+00    1
 N-E          -1.703502983016169e+01    -1.70350e+01    1
 Coulomb       4.061839981510596e+00     4.06184e+00    1
 Exchange     -4.448032885830820e-01    -4.44803e-01    1
 X-C          -1.366902465016473e+00    -1.36690e+00    1
------------------------------------------------------------
 Electronic   -7.413899258031377e+00    -7.41390e+00    1
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.413899258031377e+00    -7.41390e+00    1
============================================================


################ Wall time: 1.73478e+01 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              2.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        1.32442e-03
 Diagonalizing matrix                           3.99100e-06
 Rotating orbitals                              1.23072e-02
------------------------------------------------------------
                 Wall time: 1.36786e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       0.000000000000000e+00
 Electronic energy                   -7.465267830136265e+00
------------------------------------------------------------
 Total energy                        -7.465267830136265e+00
------------------------------------------------------------
                 Wall time: 4.74017e-01 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -2.06043e+00
 Creating operator with lambda                 -1.41549e-01
 Creating operator with lambda                 -2.05047e+00
------------------------------------------------------------
                 Wall time: 1.71244e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
 Potential part                                 4.50333e-01
 Matrix part                                    7.71000e-06
------------------------------------------------------------
   0       1.53589e+00    392            0.00000e+00      0
   1       2.58236e-01    712            0.00000e+00      0
   2       1.53366e+00    392            0.00000e+00      0
------------------------------------------------------------
                 Wall time: 4.62906e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    9.97142e-01   328    0.00000e+00     0   1.65761e+00
  1    9.96684e-01   264    0.00000e+00     0   1.67323e+00
  2    9.97351e-01   328    0.00000e+00     0   1.64104e+00
------------------------------------------------------------
                 Wall time: 4.97198e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.22700e-06
 Copy orbitals                                  5.88536e-03
 Copy orbital updates                           5.88765e-03
------------------------------------------------------------
                 Wall time: 1.18462e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     840      9.15850e-02
 Coulomb density                       392      8.48759e-02
 Coulomb potential                     200      1.00503e+00
 Coulomb grid size                     392           677376
 Coulomb grid change                     0                0
 XC alpha density                      392      5.53664e-02
 XC beta density                       392      3.30274e-02
 XC evaluate xcfun                    3528      4.63433e+00
 XC energy                             392      2.42280e-05
 XC potential                          784      6.90446e-02
 XC grid size                          392           594432
 XC grid change                          0                0
 Hartree-Fock exchange                1112      3.65900e+00
------------------------------------------------------------
                 Wall time: 9.67017e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.05580e+00   3.71913e-03        328    a    1    1
  1  -1.41004e-01   1.67120e-02        264    a    1    1
  2  -2.04559e+00   3.64248e-03        328    b    1    1
------------------------------------------------------------
 Total error:                            1.75040e-02    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -4.252448736098645e+00    -6.89064e-02    1
 Kinetic       7.313605119229749e+00    -5.73912e-02    1
 N-E          -1.701405764387290e+01     2.09722e-02    1
 Coulomb       4.035721470344935e+00    -2.61185e-02    1
 Exchange     -4.414733051862348e-01     3.32998e-03    1
 X-C          -1.359063470651811e+00     7.83899e-03    1
------------------------------------------------------------
 Electronic   -7.465267830136265e+00    -5.13686e-02    1
 Nuclear       0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.465267830136265e+00    -5.13686e-02    1
============================================================


################ Wall time: 1.68405e+01 sec ################


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    6.98151e-02    -7.413899258031377e+00   -7.41390e+00
  2    1.75040e-02    -7.465267830136265e+00   -5.13686e-02
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        1.35048e-03
 Diagonalizing matrix                           4.55300e-06
 Rotating orbitals                              1.54682e-02
------------------------------------------------------------
                 Wall time: 1.68670e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     840      9.31935e-02
 Coulomb density                       392      9.94551e-02
 Coulomb potential                     200      9.88864e-01
 Coulomb grid size                     392           677376
 Coulomb grid change                     0                0
 XC alpha density                      392      6.46332e-02
 XC beta density                       392      3.75884e-02
 XC evaluate xcfun                    3528      4.63041e+00
 XC energy                             392      2.56110e-05
 XC potential                          784      7.13433e-02
 XC grid size                          392           594432
 XC grid change                          0                0
 Hartree-Fock exchange                1112      3.58591e+00
------------------------------------------------------------
                 Wall time: 9.61356e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 5.02030e-01 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-2.05580e+00  0.00000e+00  0.00000e+00
 0.00000e+00 -1.41004e-01  0.00000e+00
 0.00000e+00  0.00000e+00 -2.04559e+00
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    a    1                      -2.055800411423594e+00
    1    a    1                      -2.045588029074528e+00
    2    b    1                      -1.410039847199747e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -4.242392425218096e+00
 Kinetic energy:                      7.381244398002861e+00
 E-N energy:                         -1.708944065807544e+01
 Coulomb energy:                      4.048664269011999e+00
 Exchange energy:                    -4.427794962991273e-01
 X-C energy:                         -1.363617361354329e+00
 El. External field energy:           0.000000000000000e+00
 Nuc. External field energy:          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -7.465928848714039e+00
 Nuclear energy:                      0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.465928848714039e+00
                    (kJ/mol)         -1.960179335524575e+04
                    (kcal/mol)       -4.684941047448327e+03
                    (eV)             -2.031582646582609e+02
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 5.495868e+01                ***
***                                                      ***
************************************************************


